app/gui/src/model/undo_redo.rs

```diff
@@ -130,7 +130,7 @@ impl Drop for Transaction {
     fn drop(&mut self) {
         if let Some(urm) = self.urm.upgrade() {
             if !self.ignored.get() {
-                info!("Transaction '{}' will create a new frame.", self.name());
+                warn!("Transaction '{}' will create a new frame. {}", self.name(), backtrace());
```

Comment:
Should this really be a warning?

Reply:
Good catch, it meant to be temporary for debugging,